meta-facebook/sb-rb/src/platform/plat_version.h

```diff
@@ -28,7 +28,7 @@
 #define DEVICE_REVISION 0x80
 
 #define FIRMWARE_REVISION_1 GET_FW_VERSION1(BOARD_ID, PROJECT_STAGE)
-#define FIRMWARE_REVISION_2 0x01
+#define FIRMWARE_REVISION_2 0x02
```

Comment:
The title and PR description says 20252901 but I see the revision 0x02 here ?

Reply:
Hi, this variable represents the release number for this stage.
If you want to see the version 20252901, you can refer to the following variables:
BIC_FW_YEAR_MSB
BIC_FW_YEAR_LSB
BIC_FW_WEEK
BIC_FW_VER